src/test/java/org/codehaus/mojo/versions/utils/BaseMojoTestCase.java

```diff
@@ -51,8 +59,39 @@
  * </pre>
  * @author Andrzej Jarmoniuk
  */
+@SuppressWarnings( "deprecation" )
```

Comment:
Why?

Reply:
Lots of MavenSettingsBuilder, DefaultArtifactRepository, etc. But I guess these warnings are useful for tracking technical debt. Removed.